src/bot/modules/reminders.ts

```diff
@@ -0,0 +1,164 @@
+              }
+            },
+            approvals: true,
+            disapprovals: true
+          }
+        },
+        approvals: true,
+        disapprovals: true
+      }
+    });
+
+    const guild = this.client.guilds.cache.get(guildId);
+    const emojisGuild = this.client.guilds.cache.get(emojisGuildId)!;
+    if (!guild) return;
+
+    await guild.members.fetch();
+
+    const role = guild.roles.cache.get(parrainRole);
+    if (!role) return;
+
+    const godfathersEmojis = await Promise.all(role.members.map((member) => getGodfatherEmoji(emojisGuild, member!)));
+
+    // Remap proposals with godfathers acceptable decisions
+    const entries: Array<{ proposal: Proposal; members_ids: Snowflake[] }> = [];
+    for (const proposal of proposals) {
+      if (proposal.type === ProposalType.SUGGESTION) {
+        if (proposal.corrections[0]) continue;
+      } else {
+        const lastCorrection = proposal.suggestion?.corrections[0];
+        if (lastCorrection && lastCorrection.id !== proposal.id) continue;
+      }
+
+      const members_ids: Snowflake[] = [
+        ...role.members
+          .filter((member) => {
+            if (proposal.approvals.some((approval) => approval.user_id === member.id)) return false;
+            if (proposal.disapprovals.some((disapproval) => disapproval.user_id === member.id)) return false;
+            if (proposal.user_id === member.id) return false;
+
+            return true;
+          })
+          .keys()
+      ];
+
+      if (!members_ids.length) continue;
+
+      entries.push({ proposal, members_ids });
+    }
+
+    // Reduce previous data into pages with a length < 4096 (max of embed description size)
+    const { pages } = entries.reduce<{ current: string; pages: string[] }>(
+      (acc, { proposal, members_ids }, index, array) => {
+        const godfathers = members_ids
+          .map((members_id) => godfathersEmojis.find(({ id }) => members_id === id)?.emoji)
+          .filter((e) => e)
+          .join(' ');
+        const line = `[${proposal.type.toLowerCase()}](https://discord.com/channels/${guild.id}/${
+          proposal.type === ProposalType.SUGGESTION ? suggestsChannel : correctionsChannel
+        }/${proposal.message_id}) ${godfathers}\n`;
+
+        if (line.length + acc.current.length > 4090) {
+          acc.pages.push(acc.current);
+          acc.current = '>>> ';
+        }
+
+        acc.current += line;
+
+        if (array.length === index + 1) acc.pages.push(acc.current);
+
+        return acc;
+      },
+      { current: '>>> ', pages: [] }
+    );
+
+    // Filter godfathers with a minimal of 3 acceptable decisions
+    const mentions = entries
+      .reduce((acc, { members_ids }) => {
+        for (const member_id of members_ids) {
+          const memberScore = acc.get(member_id) ?? 0;
+          acc.set(member_id, memberScore + 1);
+        }
+        return acc;
+      }, new Collection<Snowflake, number>())
+      .filter((score) => score > 3)
```

Comment:
Si je suis ton commentaire de 3 décisions au minimum
```suggestion
      .filter((score) => score >= 3)
```